docs/user-guide/quickstart.md

```diff
@@ -458,6 +459,94 @@ kubectl delete svc s2 -n service-2
 kubectl delete pods s2 -n service-2
 ```
 
+<a name="hrq"/>
+
+### Hierarchical resource quotas
+
+_Will demonstrate: Create and delete [HierarchicalResourceQuota](concepts.md#hierarchical-resource-quota)._
+
+Let's assume you own _acme-org_ from the previous example:
+```
+acme-org
+├── [s] team-a
+└── [s] team-b
+```
+
+You can create a HierarchicalResourceQuota in namespace `acme-org`, and the sum of 
+all subnamespaces resource usage cant go over what is configured in the hrq.
+
+Creating the hrq:
+```bash
+kubectl apply -f - << EOF
+kind: HierarchicalResourceQuota
+apiVersion: hnc.x-k8s.io/v1alpha2
+metadata:
+  name: acme-org-hrq
+  namespace: acme-org
+spec:
+  hard:
+    requests.cpu:  1
+    requests.memory: 2Gi
+    limits.cpu:  2
+    limits.memory: 4Gi
+    services: 1
+EOF
+```
+
+Lets create Service in namespace `team-a`:
+```bash
+kubectl apply -f - << EOF
+kind: Service
+apiVersion: v1
+metadata:
+  name: team-a-svc
+  namespace: team-a
+spec:
+  selector:
+    place: holder
+  ports:
+  - protocol: TCP
+    port: 80
+    targetport: 80
+EOF
+```
+
+And when we try to create Service in namespace `team-b`:
+```bash
+kubectl apply -f - << EOF
+kind: Service
+apiVersion: v1
+metadata:
+  name: team-b-svc
+  namespace: team-b
+spec:
+  selector:
+    place: holder
+  ports:
+  - protocol: TCP
+    port: 80
+    targetport: 80
+EOF
+```
+We get an error:
+```
+Error from server (Forbidden): error when creating "STDIN": admission webhood "resourcesquotasstatus.hnc.x-k8s.io" denied the request: exceeded hierarchical quota in namespace "acme-org": "acme-org-hrq", requested: services=1, used: services=1, limited: services=1
```

Comment:
Can you add newlines to make this easier to read?

Reply:
sure!
i was struggling should i make the error look as is, or should i make it more clear